### ISSUE TITLE
Format CSS grid-template-areas

### ISSUE THREAD

**Issue by aaronccasanova** [prettier/prettier#12631]
Hello 👋  I'd like to propose adding opinionated formatting rules for CSS `grid-template-areas` where column tracks are aligned to better visualize the grid.

I've created the [prettier-plugin-css-grid](https://www.npmjs.com/package/prettier-plugin-css-grid) as a proof of concept and temporary solution for the community. However, it's important to note I leveraged the [plugin.parser.preprocess](https://github.com/aaronccasanova/prettier-plugin-css-grid/blob/b2c19168fd8724019bcb6148fe60288f12099085/src/index.ts#L21-L30) method to format `grid-template-areas` (after learning Prettier leaves the values as authored) which I'm not entirely sure is valid or recommended.

I have little familiarity with the Prettier internals/parser but would be happy to contribute if a consensus is reached to move forward with the proposal. Thoughts?

**Prettier 2.6.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACVBzATgSwCYC08AtgA4A2AhvAZVnJQM5KoA6I0cqMA7hN324ALelxgi4YiXHaso6BYqXL07Xv3XDRWyTq6dZ8lceVrpei5wEa+7ANwgANB1Iwc0RslB0sEHgAU6BE8USnIeSgBPT2cAIyxKMABrOBgAZUpiOAAZHCg4ZAAzMMY4OITk1LTSRLyMZBgsAFcykDhiWLg8PC7syigMJsoMOAAxCCxiajcB5BBKJpgIJxAhGGJyAHUhHHhGGrA4NODdnAA3Xci5sEYYkDzSrBh-BIwpopLWgCtGAA80urkOAARSaEHgH3IpWcNSwjzmjBud1IuFgm3w4mQAA4AAww3ylTYJUhzFFwR5nArOACOYPgLwgpBC80YBHyXS6K3otJw9Bew3eSGKUNapWIOAazVFgJBdIKQs+zhglFi6LwmKQACYlQkcOQ6gBhCDEQVtRgAVhWTVKABUVSFhdCQGcWgBJKA9WBpMC4VwAQQ9aRgkSBkNKAF9w0A)

```sh
--parser scss
```

**Input:**

```scss
  grid-template-areas: "one two two three three three"
                      "two two three three three one"
                      "three three three one two two";
```

**Output:**

```scss
grid-template-areas:
  "one two two three three three"
  "two two three three three one"
  "three three three one two two";
```

**Expected behavior:**
```scss
grid-template-areas:
  "one   two   two   three three three"
  "two   two   three three three one  "
  "three three three one   two   two  ";
```

**Comment by alexander-akait** [comment]
I don't think it is task of prettier, you change AST of string when you add white spaces, yes, in your case it is safely, but in general context, it is unsafe modification AST node, i.e. string, just add white spaces in your code you will get what you want
